### DISCUSSION TITLE
custom connetors error

### DISCUSSION THREAD

**Discussion by zouhangwithsweet** [wevm/wagmi#3888] [Q&A]
- Vite 5.2.11   
- node 20.10.0

`Uncaught (in promise) TypeError: MetaMaskSDK is not a constructor`

```ts
import { createConfig, http } from 'wagmi'
import { mainnet, sepolia } from 'wagmi/chains'
import { metaMask } from 'wagmi/connectors'

export const config = createConfig({
  chains: [mainnet, sepolia],
  transports: {
    [mainnet.id]: http(),
    [sepolia.id]: http(),
  },
  connectors: [
    metaMask({
      injectProvider: true,
      extensionOnly: true,
    }),
  ],
})
```

<img width="619" alt="image" src="https://github.com/wevm/wagmi/assets/21095710/8eabd360-e971-4292-aec9-4db2c8b78b24">

**Comment by leor-w** [comment]
Yes, I have the same problem, but I did not customize it, using the latest version of the metaMask connector also throws the same error
